src/server/db/update.js

```diff
@@ -0,0 +1,19 @@
+import mysql from 'mysql';
```

Comment:
Function should have db queries be separate of server request and response objects
- Place method into attendanceHelpers.js
- query inside AttendanceQueries
- core functionality inside AttendanceModel